config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/environment/BundleEnvironmentValidator.java

```diff
@@ -103,7 +104,11 @@ private void findInBundle(Bundle bundle, String type, String name) {
                 entity = bundle.getJmsDestinations().get(name);
                 break;
             default:
-                throw new MissingEnvironmentException("Unexpected entity type " + type);
+                UnsupportedGatewayEntity unsupportedGatewayEntity = bundle.getUnsupportedEntities().get(name);
+                if(unsupportedGatewayEntity != null && type.equals(unsupportedGatewayEntity.getType())){
```

Comment:
Can you write something to log at INFO or WARNING level?

Reply:
Done